src/Symfony/Component/PropertyInfo/Tests/Fixtures/VirtualProperties.php

```diff
@@ -14,6 +14,6 @@
 class VirtualProperties
 {
     public bool $virtualNoSetHook { get => true; }
-    public bool $virtualSetHookOnly { set => $value; }
-    public bool $virtualHook { get => true; set => $value; }
+    public bool $virtualSetHookOnly { set (bool $value) {  } }
+    public bool $virtualHook { get => true; set (bool $value) {  } }
```

Comment:
I updated this fixtures because PHP does not report these properties as Virtual, see [reproducer](https://3v4l.org/WjmAR#v8.4.1)